pytorch_toolkit/ote/ote/api.py

```diff
@@ -1,86 +1,94 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
 import argparse
 
 import yaml
 
 
-def train_args_parser(template_path):
+def train_args_parser(config):
     parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter)
-    with open(template_path, 'r') as model_definition:
```

Comment:
what is a purpose of this change? I guess template.yaml content is not needed outside of this func, is not it?

Reply:
Unfortunately the exporter for mmdetection needs at `alt_ssd_export` section which requires template.yaml So, if you want I can change it back to load the template.yaml twice.

Reply:
As far as I can see you can use args.config and usage of `template_cfg` is redundant, please correct me if I wrong

Reply:
Ok, I'll revert all changes. But for information it breaks S principle for `train_args_parser`, 'test_args_parser', 'export_args_parser' functions from SOLID...

Reply:
Done.